### ISSUE TITLE
[clsim] remove mcTreeIndex from private/clsim/dom/I3PhotonToMCPEConverter.cxx (Trac #1492)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1484]
The mcTreeIndex is no longer needed, since the new tree does indexing just fine.

Migrated from https://code.icecube.wisc.edu/ticket/1492
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "The mcTreeIndex is no longer needed, since the new tree does indexing just fine.", 
    "reporter": "david.schultz", 
    "cc": "olivas", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "cmake", 
    "summary": "[clsim] remove mcTreeIndex from private/clsim/dom/I3PhotonToMCPEConverter.cxx", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2015-12-14T23:50:47", 
    "milestone": "", 
    "owner": "claudio.kopper", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160119T14:25:07: 
* claudio.kopper changed milestone from "IceSim 5.0" to "IceSim 5.1"
* claudio.kopper commented:

> This works just fine the way it is right now (please correct me if this is not the case). Postponing this to icesim 5.1 and lowering priority.


* claudio.kopper changed priority from "critical" to "minor"

**Comment by icecube-trac** [comment]
Trac update at 20160120T00:22:24: 
* benedikt.riedel changed status from "new" to "closed"
* benedikt.riedel commented:

> In changeset 141050:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141050"
> closes #1484
> ```


* benedikt.riedel changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas changed milestone from "IceSim 5.1" to ""
* olivas commented:

> Milestone IceSim 5.1 deleted
